projects/core/components/fullscreen/fullscreen.component.ts

```diff
@@ -0,0 +1,72 @@
+import {DOCUMENT} from '@angular/common';
+import {
+    ChangeDetectionStrategy,
+    Component,
+    ElementRef,
+    EventEmitter,
+    inject,
+    Input,
+    Output,
+    signal,
+    ViewChild,
+} from '@angular/core';
+import {TuiRoot} from '@taiga-ui/core/components/root';
+
+@Component({
+    standalone: true,
+    selector: '[tuiFullscreen]',
+    imports: [TuiRoot],
+    template: '<tui-root #root><ng-content /></tui-root>',
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    host: {
+        '(document:fullscreenchange)': 'closedByEscape($event)',
+    },
+})
+export class TuiFullscreen {
+    @ViewChild('root', {read: ElementRef})
```

Comment:
nit: no need for #root/'root', you can just write `TuiRoot`

Reply:
fixed